File: benchmark/java/src/org/slowcoders/rtgc/test/GCTest.java
package org.slowcoders.rtgc.test;

public class GCTest {

	public static void main(String args[]) throws Exception {
        StressTest.Options options = new StressTest.Options();
        options.basePayload = Integer.parseInt(args[0]); 
        options.primitiveBytes = Integer.parseInt(args[1]); 
        options.circularRefPercent = Integer.parseInt(args[2]);
        options.replacePercent = Integer.parseInt(args[3]);
        StressTest.doTest(options);
    }
}

File: benchmark/cpp-2/src/GCTest.cpp
#include "GCTest.hpp"

int main(int argc, const char* args[]) {

    StressTestOptions options;
    options.basePayload = atoi(args[1]); 
    options.primitiveBytes = atoi(args[2]); 
    options.circularRefPercent = atoi(args[3]);
    options.replacePercent = atoi(args[4]);

    testNoGCPerformance(&options);
    return 0;
}

